src/main/features/offer/index.ts

```diff
@@ -20,6 +21,10 @@ function requestHandler (): express.RequestHandler {
 
 export class Feature {
   enableFor (app: express.Express) {
+    if (app.settings.nunjucksEnv && app.settings.nunjucksEnv.globals) {
```

Comment:
This pattern allows modules to register variable with Nunjucks. We can create base class which makes it slightly more convenient and apply in parts of the app. Then Nunjucks module does not have to be overload with registering all elements of the app.